tests/test_validate_edf.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
+
+from electos.ballotmaker import validate_edf
```

Comment:
👆🏼 Ditto for comment about importing `read_edf` above.

Reply:
Agreed and fixed